src/Instruction.js

```diff
@@ -0,0 +1,14 @@
+export const Instruction = ({ userRollDice }) => {
+  if (userRollDice) {
+    return (
+      <img
+        src="https://www.animatedimages.org/data/media/710/animated-dice-image-0085.gif"
+        border="0"
+        alt="animated-dice-image-0085"
+        className="animated-gif"
+      />
+    );
+  } else {
+    return <p>Click [Continue] button...</p>;
+  }
```

Comment:
```suggestion
return userRollDice ? <img
        src="https://www.animatedimages.org/data/media/710/animated-dice-image-0085.gif"
        border="0"
        alt="animated-dice-image-0085"
        className="animated-gif"
      /> : <p>Click [Continue] button...</p>
```

Reply:
or get rid of the else statement for better readability

```suggestion
  if (userRollDice) return (
      <img
        src="https://www.animatedimages.org/data/media/710/animated-dice-image-0085.gif"
        border="0"
        alt="animated-dice-image-0085"
        className="animated-gif"
      />
    );
  return <p>Click [Continue] button...</p>;
```